### PR TITLE
With QA fixes

### PR DESCRIPTION
**Pull request by bbonanno**

### CODE REVIEW

File: packages/foundry/contracts/interfaces/IWitch.sol [hunk 1]
```diff
@@ -6,106 +6,156 @@ import "./ICauldron.sol";
...
-    function auctions(bytes12) external returns (DataTypes.Auction memory);
+    /// @dev Queries the ongoing auctions
+    /// @param vaultId Id of the vault to query an auction for
+    /// @return auction_ Info associated to the auction
+    function auctions(bytes12 vaultId)
+        external
+        returns (DataTypes.Auction memory auction_);
 
-    function lines(bytes6, bytes6) external returns (DataTypes.Line memory);
+    /// @dev Queries the params that govern how much collateral is sold over time.
```
Comment (alcueca) [inline packages/foundry/contracts/interfaces/IWitch.sol:22]:
``` suggestion
    /// @dev Queries the params that govern how time influences collateral price in auctions
```

File: packages/foundry/contracts/interfaces/IWitch.sol [hunk 2]
```diff
@@ -6,106 +6,156 @@ import "./ICauldron.sol";
...
+    /// @param ilkId Id of asset used for collateral
+    /// @param baseId Id of asset used for underlying
+    /// @return limits_ Parameters that govern how much collateral of each kind can be sold at any given time.
+    function limits(bytes6 ilkId, bytes6 baseId)
+        external
+        returns (DataTypes.Limits memory limits_);
 
-    /// @dev Put an undercollateralized vault up for liquidation
-    /// @param vaultId Id of vault to liquidate
+    /// @dev Put an under-collateralised vault up for liquidation
```
Comment (alcueca) [inline packages/foundry/contracts/interfaces/IWitch.sol:38]: Americans :roll_eyes:

**Review by alcueca — APPROVED** [review]
I rephrased one thing.

File: packages/foundry/contracts/interfaces/IWitch.sol (hunk 2 above)
Comment (bbonanno) [reply packages/foundry/contracts/interfaces/IWitch.sol:38]: the hyphen thing?

**Pull request merged**